### DISCUSSION TITLE
Can the timeline component be used for showing records from remote system?

### DISCUSSION THREAD

**Discussion by srinivasan-kr** [deejay-hub/timeline-lwc#234] [Q&A] [answered]
I need to show a list of activities stored in a different system. Is is possible now?

**Comment by deejay-hub** [comment, marked as answer]
The packaged version of Time Warp on the AppExchange can support External Objects (https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_external_objects.htm) which would allow you to view information stored outside Salesforce. You would just have to be careful to only include the minimal amount of data necessary.

You can also modify the code yourself to support an Apex callout to an external system. Since all the code is here you could add the features you need then package your own version of Time Warp. Someone has included a pull request that has the necessary code to use an Apex class to get information (https://github.com/deejay-hub/timeline-lwc/pull/108)

**Reply by srinivasan-kr** [reply]
Thanks you!
